.github/workflows/CI_build.yml

```diff
@@ -0,0 +1,235 @@
+name: CI_build
+
+on: [push, pull_request]
+
+jobs:
+
+  # failing with D:\a\AusweisApp\AusweisApp\test\helper\common\MockReleaseInformation.cpp(42,35): error C2248: 'QString::QString': cannot access private member declared in class 'QString' [C:\_build\test\helper\common\AusweisAppTestHelperCommon.vcxproj]
+  #build_windows:
+
+
+    # runs-on: windows-latest
+    # strategy:
+      # fail-fast: false
+      # matrix:
+        # build_configuration: [Release]
+        # build_platform: [x64]
+
+    # steps:
+
+    # - name: Install openssl dev
+      # run: |
+           # choco install openssl
+           # dir "C:\Program Files"
+
+    # - name: Checkout repo
+      # uses: actions/checkout@v4
+
+    # - name: Add msbuild to PATH
+      # uses: microsoft/setup-msbuild@v1
+
+    # - name: Install Qt
+      # uses: jurplel/install-qt-action@v3
+      # with:
+        # version: '6.5.*'
+        # modules: 'qtscxml qtwebsockets qtshadertools qtconnectivity'
+        # setup-python: 'false'
+
+    # - name: generate cmake
+      # run: |
+           # mkdir c:\_build
+           # cd c:\_build
+           # cmake -G "Visual Studio 17 2022" -A ${{ matrix.build_platform }} -T "v143" D:\a\AusweisApp\AusweisApp
+
+    # - name: build cmake
+      # run: |
+           # cd c:\_build
+           # cmake --build . --config ${{ matrix.build_configuration }} --target package
+
+    # - name: run ctest
+      # run: |
+           # cd c:\_build
+           # ctest --output-on-failure -C "${{ matrix.build_configuration }}"
+
+  # ubuntu 22.04 comes just with QT 6.2.4 and Qt >= 6.4 is required
+  build_linux_cmake:
+
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        build_configuration: [Release]
+        build_platform: ["Unix Makefiles"]
+
+    steps:
+    - uses: actions/checkout@v4
+
+    - name: Install packages via apt
+      run: |
+           sudo apt update -qq && sudo apt install -y cmake pkg-config libssl-dev libudev-dev libhttp-parser-dev libpcsclite-dev libgl1-mesa-dev qt6-l10n-tools
+
+    - name: Install Qt
+      uses: jurplel/install-qt-action@v3
+      with:
+        version: '6.5.*'
+        modules: 'qtscxml qtwebsockets qtshadertools qtconnectivity'
+        setup-python: 'false'
```

Comment:
is it possible to use the built libs from below as another matrix job?

Reply:
@misery I added another workflow which combines the build of the libs and the direct use of it for the app. I think with that the extra workflow of the lib build is obsolete. What do you think.
The build of the app with pre build QT installations might be still of some interest for testing of newer QT versions..

Reply:
I used unix makefiles for android and ios as with ninja build the error:
```

Run cmake --build _build_libs --config Release
ninja: error: build.ninja:313: multiple rules generate download/qt-everywhere-src-6.6.1.tar.xz [-w dupbuild=err]
```
appears.

Reply:
Yes, if we use a combined build there is no need for a separate lib build job.

Strange error. Did you just Multi-Config Ninja Genereator somewhere? Maybe you could remove --config Release as it is needed for Multi-Config only.